Gemfile.lock

```diff
@@ -9,7 +9,9 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.6.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```

Comment:
@prakharrathi25 Any idea why two eventmachine's are there?

Reply:
I think one of them has been added because of my vscode version. I am not messing with the `.lock` file because it is required for proper runtime of the app currently.